libs/SmartSync/SmartSync/Classes/Util/SFParentChildrenSyncHelper.m

```diff
@@ -64,12 +66,11 @@ + (NSString*) getNonDirtyRecordIdsSql:(SFParentInfo*)parentInfo childrenInfo:(SF
 }
 
 + (void)saveRecordTreesToLocalStore:(SFSmartSyncSyncManager *)syncManager target:(SFSyncTarget *)target parentInfo:(SFParentInfo *)parentInfo childrenInfo:(SFChildrenInfo *)childrenInfo recordTrees:(NSArray *)recordTrees {
-
+    [SFSDKAppFeatureMarkers registerAppFeature:kSFAppFeatureRelatedRecords];
```

Comment:
If we don't care about how often they sync up or sync down related records, shouldn't we simply put the registerAppFeature call in the constructor for parent-sync-down-target and parent-sync-up-target?